### DISCUSSION TITLE
Multijugador

### DISCUSSION THREAD

**Discussion by Yulyisabel** [mupen64plus-ae/mupen64plus-ae#1119] [General]
La app dice que no puede generar el código para el modo multijugador

**Comment by fzurita** [comment]
In order to get a code:

 You have to be connected WiFi.

 Your router has to be UPnP compatible, not all routers are compatible.

 Your Internet service has to not block servers .

 If any of these is not true, then you will not be able to get a code.

 If the two people are on the same WiFi, it is not necessary to get a code. The server phone will automatically show in the client and if not, you can use an IP address.

**Comment by benjaJajala** [comment]
Tengo un problema esque cuando juego El Smash Bros 64 con un amigo a el siempre esta su pantalla en el "pasado" como atrasada a los que veo yo , y eso hace que sea injugable jugar con el

**Reply by fzurita** [reply]
Eso pasa si el celular de el no es suficientemente rápido para correr el juego a 60fps.

**Reply by benjaJajala** [reply]
Si, creo que tenes razon , el tiene un celu muy viejo ,gracias
